packages/flutter_tools/lib/src/commands/trace.dart

```diff
@@ -16,14 +16,18 @@ import '../tracing.dart';
 class TraceCommand extends FlutterCommand {
   TraceCommand() {
     requiresPubspecYaml();
-    argParser.addFlag('start', negatable: false, help: 'Start tracing.');
-    argParser.addFlag('stop', negatable: false, help: 'Stop tracing.');
-    argParser.addOption('out', help: 'Specify the path of the saved trace file.');
-    argParser.addOption('duration',
-        defaultsTo: '10', abbr: 'd', help: 'Duration in seconds to trace.');
     argParser.addOption('debug-port',
```

Comment:
It's too bad that this is called `--debug-port` here and `--observatory-port` in `flutter screenshot`...

Reply:
yeah. i'll leave that to another yak shave.